modules/bq-log-alerting/README.md

```diff
@@ -0,0 +1,146 @@
+```
+
+The **source_name** format is `organizations/<ORG_ID>/sources/<SOURCE_ID>`.
+
+**Note 2:** The module has a **dry_run** optional mode (`dry_run = true`). In this mode, instead of creating the finding in Security Command Center the module writes the finding to Google logging.
+
+## Requirements
+
+The following sections describe the requirements which must be met in
+order to invoke this module.
+
+### General
+
+* You need an existing "logging" project.
+* A [Log export](https://github.com/terraform-google-modules/terraform-google-log-export) with a [BigQuery destination](https://github.com/terraform-google-modules/terraform-google-log-export/tree/master/modules/bigquery) in the logging project. The export filter should include at least:
+  * "logName: /logs/cloudaudit.googleapis.com%2Factivity"
+  * "logName: /logs/cloudaudit.googleapis.com%2Fdata_access"
+  * "logName: /logs/compute.googleapis.com%2Fvpc_flows"
+* It is necessary to use a Service Account to authenticate the Google Terraform provider to be able to create the Security Command Center "BQ Log Alerts" Source.
+This is a restriction of the Security Command Center API
+* [Google App Engine](https://cloud.google.com/appengine) must be enabled in the logging project. To enable it manually use:
+
+```shell
+gcloud app create \
+--region=<REGION> \
+--project=<LOGGING_PROJECT> \
+--impersonate-service-account=<TERRAFORM_SERVICE_ACCOUNT_EMAIL>
+```
+
+**Note:** The selected region cannot be changed after creation.
+
+### IAM Roles
+
+The Service Account which will be used to invoke this module must have the following IAM roles:
+
+* Project level:
+  * Cloud Functions Developer: `roles/cloudfunctions.developer`
+  * Storage Admin: `roles/storage.admin`
+  * Pub/Sub Admin: `roles/pubsub.admin`
+  * Service Account User: `roles/iam.serviceAccountUser`
+* Organization level
+  * Security Center Sources Editor: `roles/securitycenter.sourcesEditor`
+  * Security Admin: `roles/iam.securityAdmin`
+
+You you are deploying this module in the logging project of the Terraform Example Foundation using the Terraform Service account created in the Foundation it already has all the necessary permissions in the logging project.
+
+### APIs
+
+The project against which this module will be invoked must have the
+following APIs enabled:
+
+* App Engine Admin API: `appengine.googleapis.com`
+* BigQuery API: `bigquery.googleapis.com`
+* Cloud Build API: `cloudbuild.googleapis.com`
+* Cloud Functions API: `cloudfunctions.googleapis.com`
+* Cloud Logging API: `logging.googleapis.com`
+* Cloud Pub/Sub API: `pubsub.googleapis.com`
+* Cloud Scheduler API: `cloudscheduler.googleapis.com`
+* Cloud Storage API: `storage-component.googleapis.com`
+* Security Command Center API: `securitycenter.googleapis.com`
+
+### Software Dependencies
+
+* [Terraform][terraform-site] v0.12
+* [Terraform Provider for Google Cloud Platform][terraform-provider-gcp-site] v2.5
```

Comment:
Let's cap this to `3.25.0`. Function deployments tended to be flaky before a retry was added.